.drone.star

```diff
@@ -751,6 +755,9 @@ def settingsUITests(ctx, storage = "ocis", accounts_hash_difficulty = 4):
                     "cd /srv/app/web",
                     "git checkout $WEB_COMMITID",
                     "yarn install --immutable",
+                    "yarn build",
```

Comment:
@individual-it not sure if we need to build `web` here btw, just added it since it _seemed like it was missing_ yet tests passed before without problems apparently